components/browser/errorpages/src/test/java/mozilla/components/browser/errorpages/ErrorPagesTest.kt

```diff
@@ -112,10 +112,12 @@ class ErrorPagesTest {
 
         val htmlFilename = "htmlResource.html"
 
+        val uri = "sampleUri"
+
         val errorPage = createUrlEncodedErrorPage(
             context,
             errorType,
-            null,
+                uri,
```

Comment:
nit: extra spacing

Reply:
My AS was set to use continuation index on function calls on this project. I'm not sure why 🤔